proto/internal/dispatch/v1/dispatch.proto

```diff
@@ -14,12 +14,19 @@ service DispatchService {
 }
 
 message DispatchCheckRequest {
+  enum DebugSetting {
+    NO_DEBUG = 0;
+    ENABLE_DEBUGGING = 1;
+  }
+
   ResolverMeta metadata = 1 [ (validate.rules).message.required = true ];
 
   core.v1.ObjectAndRelation resource_and_relation = 2
       [ (validate.rules).message.required = true ];
   core.v1.ObjectAndRelation subject = 3
       [ (validate.rules).message.required = true ];
+
+  DebugSetting debug = 4;
```

Comment:
should this be in resolvermeta?

Reply:
No, otherwise we'd have to clear it explicitly from there which made things a bit more complicated when cloning